vision/fiducial_detection_and_pose.hh

```diff
@@ -24,6 +24,11 @@ struct MarkerInWorld {
   int id;
 };
 
+struct board_ids_and_corners {
```

Comment:
Conform to styleguide: CamelCase for types

Reply:
*self flagellates*